docs/user-guide/limitations.md

```diff
@@ -1,15 +1,45 @@
 # Sysbox User Guide: Functional Limitations
 
 This document describes functional restrictions and limitations of Sysbox and
-system containers.
+the containers created by it (i.e., system containers).
```

Comment:
You use "sysbox container" in the rest of the doc, so, for consistency, let's do the same here ...

Reply:
Done.